collections/nemo_nlp/nemo_nlp/data/datasets/token_classification.py

```diff
@@ -144,22 +140,22 @@ def get_features(queries,
 
         all_segment_ids.append([0] * max_seq_length)
 
-    logger.info(f'{too_long_count} are longer than {max_seq_length}')
+    nemo.logging.info(f'{too_long_count} are longer than {max_seq_length}')
```

Comment:
shouldn't that be .warning instead of .info?

Reply:
done